src/journal/models.py

```diff
@@ -157,10 +169,9 @@ def get_setting(self, group_name, setting_name):
         return setting_handler.get_setting(group_name, setting_name, self, create=False).processed_value
 
     @property
-    @cache(300)
     def name(self):
         try:
-            return setting_handler.get_setting('general', 'journal_name', self, create=False, fallback='en').value
+            return setting_handler.get_setting('general', 'journal_name', self, create=False).value
```

Comment:
Doesn't this raise an attribute error now? Also create is deprecated and I think we want default=True

Reply:
Fixed the AttributeError already but yes swapped create for default.